packages/next/client/image.tsx

```diff
@@ -698,15 +698,21 @@ export default function Image({
       }
     }
   }
-
   const imgStyle = Object.assign({}, style, layout === 'raw' ? {} : layoutStyle)
+  const svgBlurPlaceholder = `url("data:image/svg+xml;charset=utf-8,%3Csvg xmlns='http%3A//www.w3.org/2000/svg' xmlns%3Axlink='http%3A//www.w3.org/1999/xlink' viewBox='0 0 ${widthInt} ${heightInt}'%3E%3Cfilter id='b' color-interpolation-filters='sRGB'%3E%3CfeGaussianBlur stdDeviation='50'%3E%3C/feGaussianBlur%3E%3CfeComponentTransfer%3E%3CfeFuncA type='discrete' tableValues='1 1'%3E%3C/feFuncA%3E%3C/feComponentTransfer%3E%3C/filter%3E%3Cimage filter='url(%23b)' x='0' y='0' height='100%25' width='100%25' xlink%3Ahref='${blurDataURL}'%3E%3C/image%3E%3C/svg%3E");`
```

Comment:
`xlink:href` is deprecated

The MDN docs recommend `href` instead, or rather both for backwards compat

https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/xlink:href

Reply:
I'm a little unsure about doing both, as the blurDataURL can be really long. Even if it compresses out since it's duplicated, it still makes the image elements look huge and very confusing in the inspector and html. According to caniuse it looks pretty safe to use just href (https://caniuse.com/mdn-svg_elements_use_href). Should we stick with just that?

Reply:
Yeah lets use `href` 👍